redpanda/templates/configmap.yaml

```diff
@@ -30,6 +30,21 @@ metadata:
   {{- toYaml . | nindent 4 }}
 {{- end }}
 data:
+  bootstrap.yaml: |
+{{- if (include "redpanda.semver" . | semverCompare ">=22.1.4") }}
+{{- if .Values.auth.sasl.enabled }}
+      enable_sasl: true 
+  {{- with .Values.auth.sasl.users }}
+      superusers: ['{{ (first .).name }}']
+  {{- end }}
+{{- end }}
+{{- end }}
+ {{- with .Values.config.cluster }}
+    {{- toYaml . | nindent 6 }}
+  {{- end }}
+  {{- with .Values.config.tunable }}
+    {{- toYaml . | nindent 6 }}
+  {{- end }}
```

Comment:
It seems that `bootstrap.yaml` will be configured here with values from `values.yaml` regardless of the Redpanda version. I would expect to be able to use Redpanda `21.x` by modifying the version, and then the helm chart add the config to `redpanda.yaml` instead `bootstrap.yaml`.

Reply:
@vuldin - I literally just added that question to the PR comments. It should be possible to alter the config map so depending on version either updates the configmap for redpanda.yaml or not. I will look into it now.

Reply:
Commit [b6e8ff8](https://github.com/redpanda-data/helm-charts/pull/117/commits/b6e8ff8e5ded707999865a629155e5ae8394e2c3) should now allow the old style values.yaml to be compatible with the older versions.

Reply:
Discussion continued [here](https://github.com/redpanda-data/helm-charts/pull/117#discussion_r929542045)